### PR TITLE
Create script to enqueue build job logs

### PR DESCRIPTION
**Pull request by Jullianos**

### CODE REVIEW

**Review by vitalied — CHANGES_REQUESTED** [review]
@Jullianos @AndriiMysko pls use `rubocop -A`

**Comment by Jullianos** [comment]
> Specs are missing
The test cases for this service should be exactly the same as they are for the `queue_processing_logs_service`. I've created the `manually_schedule_logs_service_spec`.

**Review by vitalied — CHANGES_REQUESTED** [review]
@Jullianos @AndriiMysko pls use `rubocop -A` and fix all the issues before submitting the code for review

File: lib/scripts/scheduler.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/ruby
```
Comment (vitalied) [inline lib/scripts/scheduler.rb:1]: @Jullianos pls move this to a rake task.
Ex: https://stackoverflow.com/questions/28110384/how-to-parse-rake-arguments-with-optionparser

**Review by vitalied — CHANGES_REQUESTED** [review]

**Comment by Jullianos** [comment]
@vitalied please ignore the last commit I'm reworking the script from scratch.

File: lib/scripts/scheduler.rb (hunk 1 above)
Comment (Jullianos) [reply lib/scripts/scheduler.rb]: I've created a scheduler.rake task, and this is the way you call the rake command:
log_ids: `> rake schedule_logs -- '--ids=1,24,55'`
time_frame: `> rake schedule_logs -- '--from=02-02-2022' '--to=03-03-2022'`

File: app/services/manually_schedule_logs_service.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return if log_ids.empty?
+
+    Log.where(id: log_ids).update_all(scan_status: :queued, scan_status_updated_at: Time.zone.now)
+    ScanTrackerEntry.create_entries(log_ids, :queued)
+    start_process_log_batch_job(log_ids)
+  end
+
+  def start_process_log_batch_job(log_ids)
+    log_ids.each_slice(Settings.queue_limit) do |log_ids_chunk|
+      ProcessLogBatchJob.set(priority: Settings.low_queue_priority, queue: :low_priority_logs_for_scanning)
```
Comment (vitalied) [inline app/services/manually_schedule_logs_service.rb:46]: @Jullianos @AndriiMysko there is no such `ProcessLogBatchJob` class

**Review by vitalied — CHANGES_REQUESTED** [review]

**Pull request merged**